packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,146 @@
+import { html, css, LitElement } from 'lit-element';
+import { LazyLoaderService } from '@internetarchive/lazy-loader-service';
+import loaderIcon from './assets/icon-loader';
+import questionIcon from './assets/icon-question';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get properties() {
+    return {
+      widgetSrc: { type: String },
+      isLoading: { type: Boolean },
+      buttonVisible: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.widgetSrc = '';
+    this.buttonVisible = true;
+    this.isLoading = false;
+  }
+
+  async initiateZenDesk() {
+    this.isLoading = true;
+
+    // load third-party script
+    const lazyLoaderService = new LazyLoaderService();
+    await lazyLoaderService.loadScript({
+      attributes: [{ key: 'id', value: 'ze-snippet' }],
+      src: this.widgetSrc,
+    });
+
+    let countdownTicksLeft = 20;
+    const timeoutTimer = setInterval(() => {
+      // See if button has appeared
+      const iframe = document.getElementById('launcher');
+
+      /* istanbul ignore else */
+      if (iframe) {
+        const button = iframe.contentWindow.document.querySelector('button');
+
+        /* istanbul ignore else */
+        if (button) {
+          // Found it! Click on it and hope the window opens!
+          button.click();
+          this.buttonVisible = false;
+          clearInterval(timeoutTimer);
+          return;
+        }
+      }
+
+      // Try again later
+      countdownTicksLeft -= 1;
+      /* istanbul ignore if */
+      if (countdownTicksLeft <= 0) clearInterval(timeoutTimer);
+    }, 250);
+  }
```

Comment:
On the petabox side, I see
```
helpButton.addEventListener('ZendeskHelpButton', () => {})
```
But I don't see you emitting an event when the button gets clicked. Are you expecting to be tracking an event?